README.md

```diff
@@ -1,42 +1,69 @@
-# What's inside
+# Containerized Ansible: What's inside #
 
-This container provide ansible toolstack inside a container.
+This container provides the ansible toolstack inside a container.
 * '''Dockerfile''' with the definition of the ansible container
-** based on OpenSUSE Tumbleweed
-** installs ansible and some additional tools
-* System Setup
-** Make sure needed packages installed on the host
----
-# sudo transactional-update pkg install podman python310-rpm
----
-*** Install ansible commands wrapers
-If current user is root /usr/local/bin is updated, for non-root users
-the user's ~/bin area is updated.
----
-as root:
-# (cd ~; podman container runlabel root-install ansible)
-as non-root
-# (cd ~; podman container runlabel user-install ansible)
----
-*** Ansible Commands
----
-ansible
-ansible-community
-ansible-config
-ansible-connection
-ansible-console
-ansible-doc
-ansible-galaxy
-ansible-inventory
-ansible-playbook
-ansible-pull
-ansible-vault
----
-
-*** Uninstall ansible commands wrapers
----
-as root:
-# (cd ~; podman container runlabel root-unstall ansible)
-as non-root
-# (cd ~; podman container runlabel user-unstall ansible)
----
+  * based on OpenSUSE Tumbleweed
+  * installs ansible and some additional tools
+## System Setup ##
+* Make sure needed packages installed on the host
+  * sudo transactional-update pkg install podman python310-rpm
+  * system reboot is required after all transactional updates
+    * sudo shutdown -r now
+
+## Ansible commands
+The ansible commands are provided as symlinks to ansible-wrapper.sh.
+The commands will instanciate the container and execute the ansible
+command.
+
+## To install ansible commands ##
+
+* as root:
+  * for the root user the ansible commands are placed in /usr/local/bin
+  * (cd /; podman container runlabel install ansible)
```

Comment:
The system install should not require being cd'd to `/`... If it does we should fix that...

Reply:
```suggestion
  * podman container runlabel install ansible
```